src/topp/FileFilter.cpp

```diff
@@ -1281,8 +1281,8 @@ class TOPPFileFilter :
     }
 
     // create new experiment
-    PeakMap exp2 = (SpectrumSettings)exp; // copy meta data
-    exp2.clear(false); // clear spectra
+    PeakMap exp2;
+    exp.getExperimentalSettings() = (ExperimentalSettings)exp.getExperimentalSettings(); // copy meta data
```

Comment:
@timosachsenberg -- wait, this should be `exp2` here, not `exp`, right?

Reply:
Thanks!